packages/js-util-concurrency/src/lib/common/sleep.ts

```diff
@@ -0,0 +1,11 @@
+import { createDeferredPromise } from './deferred-promise';
+
+export async function sleep(ms: number): Promise<void> {
```

Comment:
I know you most likely overlooked it because it was only the comment for all the comments, but there should be documentation for all the exported methods.

Reply:
Documentation will be done later in one step with Typedoc or something similar.

Reply:
Doing it bit by bit is just a pain in the ass. In addition to that, there won't be a release without proper documentation anyway.